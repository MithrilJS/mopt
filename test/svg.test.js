"use strict";

var assert = require("assert"),

    code = require("./lib/code");

describe("SVG", function() {
    it("should support svg selectors", function() {
        assert.equal(
            code(`m("svg")`),
            `m.vnode("svg",undefined,undefined,[],undefined,undefined);`
        );
    });
    
    it("should support nested svg selectors", function() {
        assert.equal(
            code(`m("svg", m("g"))`),
            `m.vnode("svg",undefined,undefined,[m.vnode("g",undefined,undefined,[],undefined,undefined)],undefined,undefined);`
        );
    });
});
