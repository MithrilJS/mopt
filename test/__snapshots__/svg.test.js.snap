// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG should support nested svg selectors 1`] = `"m.vnode(\\"svg\\",undefined,undefined,[m.vnode(\\"g\\",undefined,undefined,[],undefined,undefined)],undefined,undefined);"`;

exports[`SVG should support svg selectors 1`] = `"m.vnode(\\"svg\\",undefined,undefined,[],undefined,undefined);"`;
